src/extras/synchronizer.js

```diff
@@ -173,6 +183,14 @@ function attachZoomHandlers(gs, syncOpts, prevCallbacks) {
             }
             continue;
           }
+
+          // Only redraw if there are new options
+          if (
+            arraysAreEqual(opts.dateWindow, gs[j].getOption('dateWindow')) && 
+            arraysAreEqual(opts.valueRange, gs[j].getOption('valueRange'))
+          )
```

Comment:
Style nit: add curlies for multiple line if statements.

(I try to follow https://google.github.io/styleguide/javascriptguide.xml)